src/components/WorkHistory/index.tsx

```diff
@@ -67,6 +87,7 @@ interface IWorkplace {
   description: string;
   startDate: string;
   endDate: string;
+  isOrganisation: any;
```

Comment:
You created a function for this, should know the type it's returning. Why is this still any?